### DISCUSSION TITLE
voice call and video call feature

### DISCUSSION THREAD

**Discussion by Piyush0125** [LiveHelperChat/livehelperchat#1923] [Q&A]
live helper chat voice call and video call is not responding .when I click on start the call button it redirect to blank page

**Comment by remdex** [comment]
You already asked same question on discord. As long all works here nothing I can do. There is paid support if you need personal help.

**Comment by Piyush0125** [comment]
If you give only posible reason 
I am very thankful to you

**Reply by remdex** [reply]
Invalid config and you did not followed manual... And don't know how to use chrome debug toolbar. So hire someone with more basic knowledge :)

**Comment by Piyush0125** [comment]
I done the configuration according to live helper chat documentation and also debug but there is no any error show in a chrome console to fix
Its working fine earlier but after migrating the server to cpanel then these error occur

**Reply by remdex** [reply]
There is paid support if you need one. Nothing else I can help with.

**Reply by Piyush0125** [reply]
kindly please answer me the Live helper chat has full compatibility with Cpanel?
I am very thankful to you

**Reply by remdex** [reply]
It does. Where php is hosted does not matter.

**Reply by Piyush0125** [reply]
Ok thank you

**Comment by Piyush0125** [comment]
hi remdex
I am stuck in voice and vedio call feature 
I did a proper configuration and also upgraded  to the latest version. but there is still a blank page .

kindly please suggest what is the possible reason behind these .

and also in a chat bot some times calling icons show sometime its not
![Screenshot (255)](https://user-images.githubusercontent.com/82926173/227200407-39d52fb1-0036-4a7c-84bf-af0e9d927813.png)

**Reply by remdex** [reply]
As last time there is paid support if you need help... I'm not magician to ques from blank screen what you did wrong :)
